modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java

```diff
@@ -269,6 +269,42 @@ public void testAddKBArticlesMarkdownWithSingleApproverWorkflow()
 		importMarkdownArticles();
 	}
 
+	@Test
+	public void testAddKBArticleUpdateExpirationDateUpdateStatus()
```

Comment:
I would rename it to something like `testExpireKBArticleUpdatesStatus()`

Reply:
I will push a name change to `testUpdateKBArticleExpirationDateUpdatesStatus`

Reply:
New name is better. Thanks